src/lib/__tests__/graphqlErrorHandler.test.ts

```diff
@@ -86,7 +86,7 @@ describe("graphqlErrorHandler", () => {
       it("does include a HTTP status code when a response is present", () => {
         const originalError: ServerError = {
           message: "underlying",
-          response: { status: 404 },
+          response: { status: 404 } as Response,
```

Comment:
Use type assertion here to avoid having to mock out the entire response object. Should be sufficient to just check for status for the purposes of testing.